### PR TITLE
Adds a code of conduct.

### PR DESCRIPTION
**Pull request by towanda**
Based on: http://contributor-covenant.org/

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 #### Features
 
-* Your contribution here.
```
Comment (dblock) [inline CHANGELOG.md]: Put back that line ;)

**Comment by towanda** [comment]
ouch! fixed ;)

**Comment by dblock** [comment]
I'd like this on https://github.com/ruby-grape/ruby-grape.github.io somewhere please!

**Comment by towanda** [comment]
@dblock sure :)

**Comment by towanda** [comment]
Now we are listed here: http://contributor-covenant.org/adopters/

**Comment by dblock** [comment]
Moved into ruby-grape.org in https://github.com/ruby-grape/ruby-grape.github.io/commit/da76718ff723d8e6e6a3f45972045ce133220842 and https://github.com/ruby-grape/grape/commit/d3730e52700a533df7c1a552e2b09142575716d6.

**Pull request merged**